heterogeneityEval.m

```diff
@@ -0,0 +1,145 @@
+function [dose] = heterogeneityEval(particleType, doseMode, exportFolder, patients)
+% Heterogeneity correction evaluation script
+%
+% Computes the dose for the HIT plan (recalc), for protons and for carbons
+% with and without heterogeneity correction. Saves the workspace for each
```

Comment:
Maybe you can move these files into the tools subfolder ? I think that's a better place for the files.